README.md

```diff
@@ -109,3 +110,13 @@ yarn workspace @anticrm/tool upgrade-workspace workspace
 ```bash
 yarn add -D @typescript-eslint/eslint-plugin @typescript-eslint/parser eslint eslint-config-standard eslint-plugin-import eslint-plugin-node eslint-plugin-promise eslint-plugin-standard`
 ```
+
+## Git hooks
+
+### pre-commit
+
+Create symlink to format and autofix changed files on commit:
+
+```bash
+ln -s ../../git-hooks/pre-commit .git/hooks/pre-commit
```

Comment:
../../ could not be used, since this readme is used from root and file it points are outside of repository.

Reply:
Just to resume, `../../git-hooks/pre-commit` is the target relative to `.git/hooks`, it stays same wherever you decide to create link.
For instance:
```
λ peka packages → ln -s ../../git-hooks/pre-commit ../.git/hooks
λ peka packages → ls -l ../.git/hooks/pre-commit 
lrwxrwxrwx 1 user user 26 Apr  5 21:59 ../.git/hooks/pre-commit -> ../../git-hooks/pre-commit
```

Reply:
ok